client/src/app/site/users/components/user-list/user-list.component.html

```diff
@@ -93,80 +86,82 @@ <h2 translate>Participants</h2>
-
-<mat-menu #userMultiSelectMenu="matMenu">
-    <button mat-menu-item *osPerms="'users.can_manage'" (click)="setGroupSelected(null)">
-        <mat-icon>archive</mat-icon>
-        <span translate>Set Groups</span>
-        <!-- TODO bottomsheet/menu? -->
-    </button>
-
-    <mat-divider></mat-divider>
-
-    <button mat-menu-item *osPerms="'users.can_manage'" (click)="setActiveSelected(true)">
-        <mat-icon>add_circle</mat-icon>
-        <span translate>Set Active</span>
-    </button>
-
-    <button mat-menu-item *osPerms="'users.can_manage'" (click)="setActiveSelected(false)">
-        <mat-icon>remove_circle</mat-icon>
-        <span translate>Set Inactive</span>
-    </button>
-
-    <mat-divider></mat-divider>
-
-    <button mat-menu-item *osPerms="'users.can_manage'" (click)="setPresentSelected(true)">
-        <mat-icon>add_circle</mat-icon>
-        <span translate>Set as Present</span>
-    </button>
-
-    <button mat-menu-item *osPerms="'users.can_manage'" (click)="setPresentSelected(false)">
-        <mat-icon>remove_circle</mat-icon>
-        <span translate>Set as Not Present</span>
-    </button>
-
-    <mat-divider></mat-divider>
-
-    <button mat-menu-item *osPerms="'users.can_manage'" (click)="setCommitteeSelected(true)">
-        <mat-icon>add_circle</mat-icon>
-        <span translate>Set as Committee</span>
-    </button>
-
-    <button mat-menu-item *osPerms="'users.can_manage'" (click)="setCommitteeSelected(false)">
-        <mat-icon>remove_circle</mat-icon>
-        <span translate>Unset Committee</span>
-    </button>
-
-    <mat-divider></mat-divider>
+    <div *ngIf="!isMultiSelect">
+        <button mat-menu-item *osPerms="'users.can_manage'" (click)="toggleMultiSelect()">
+            <mat-icon>library_add</mat-icon>
+            <span translate>MultiSelect</span>
+        </button>
 
-    <button mat-menu-item *osPerms="'users.can_manage'" (click)="sendInvitationSelected()">
-        <mat-icon>mail</mat-icon>
-        <span translate>Send invitations</span>
-    </button>
+        <button mat-menu-item *osPerms="'users.can_manage'" routerLink="groups">
+            <mat-icon>people</mat-icon>
+            <span translate>Groups</span>
+        </button>
 
-    <button mat-menu-item *osPerms="'users.can_manage'" (click)="deleteSelected()">
-        <mat-icon>delete</mat-icon>
-        <span translate>Delete selected</span>
-    </button>
+        <button mat-menu-item>
+            <mat-icon>save_alt</mat-icon>
+            <span translate>Import ...</span>
+        </button>
 
+        <button mat-menu-item (click)="csvExportUserList()">
+            <mat-icon>archive</mat-icon>
+            <span translate>Export as CSV</span>
+        </button>
+    </div>
+    <div *ngIf="isMultiSelect">
+        <div *osPerms="'users.can_manage'">
+            <button mat-menu-item (click)="setGroupSelected(null)">
+                <mat-icon>archive</mat-icon>
+                <span translate>Set Groups</span>
```

Comment:
only the first letter should be capitalized. groups, active, inactive with small first letter